usermods/sht/usermod_sht.h

```diff
@@ -0,0 +1,491 @@
+#pragma once
+
```

Comment:
Better to add #include <Wire.h> here, so the SHT85 driver uses the "correct" Wire driver.

Reply:
Just curious: In which cases would it not / how can situation arise that it would use the wrong one?

Reply:
Well it would be possible that the sht85 includes its "preferred" i2c Wire implementation, if it does not find the global one. But this is just a mesure of precaution.

Reply:
Update: seems not strictly needed. usermod_list.cpp includes wled. h, which already include Wire.h

https://github.com/Aircoookie/WLED/blob/b94e0ef797b0c93fc97f9c9b194ec6b2e41a1999/wled00/wled.h#L89

So you don't need to do that again.